db/migrate/20250313045010_add_index_to_users_ip_address.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
```

Comment:
Rerun of https://github.com/discourse/discourse/pull/31792/files#diff-a175d73be06060e1550229eb5cd92e24e0b048282080dd2f68e4070ff7dc58be